templates/charon.yaml

```diff
@@ -41,6 +41,30 @@ spec:
       securityContext:
         runAsUser: 1000
         fsGroup: 1000
+      volumes:
+        - name: validators
+          projected:
+            sources:
+              - secret:
+                  name: $NODE_NAME-validators
+        - name: private-key
+          projected:
+            sources:
+              - secret:
+                  name: $NODE_NAME-charon-enr-private-key
+        - name: cluster-lock
+          emptyDir: {}
+      initContainers:
```

Comment:
for the init container we need to define a service account "charon-init" to allow it to access gcp buckets.

`serviceAccountName: charon-init`

the actual service account, we need to create a similar terraform like this one https://github.com/ObolNetwork/obol-infrastructure/blob/main/gcp/modules/obol-cluster/add-ons/vmbackup-sa.tf but `charon-sa.tf` and change